engine/controllers/models.cc

```diff
@@ -351,8 +352,12 @@ void Models::ImportModel(
           modelPath, file_path,
           std::filesystem::copy_options::update_existing);
       model_config.files.push_back(file_path.string());
+      uintmax_t size = std::filesystem::file_size(file_path);
+      model_config.size = size;
     } else {
       model_config.files.push_back(modelPath);
+      uintmax_t size = std::filesystem::file_size(modelPath);
```

Comment:
`auto size` might be better? we won't need the `#include <cstdint>`

Reply:
Oh. I thought a clear type declaration would be better